private/buf/cmd/buf/command/mod/modupdate/modupdate.go

```diff
@@ -111,32 +111,13 @@ func run(
 		return err
 	}
 
-	remote := bufconnect.DefaultRemote
-	if moduleConfig.ModuleIdentity != nil && moduleConfig.ModuleIdentity.Remote() != "" {
-		remote = moduleConfig.ModuleIdentity.Remote()
-	} else {
-		for _, moduleReference := range moduleConfig.Build.DependencyModuleReferences {
-			if moduleReference.Remote() != bufconnect.DefaultRemote {
-				warnMsg := fmt.Sprintf(
```

Comment:
If it's an unnamed module, we moved this from being a warning to being a debug message. Most likely the user already knows they don't have a module name, and it's supposed to be local. The message is still there, just behind a `--debug` flag.